src/js/actions/layers.js

```diff
@@ -817,11 +817,11 @@ define(function (require, exports) {
      *
      * @param {Document} document
      * @param {Layer|Immutable.Iterable.<Layer>} layers
-     * @param {boolean} expand If true, expand the groups. Otherwise, collapse them.
-     * @param {boolean=} descendants Whether to expand all descendants of the given layers.
+     * @param {boolean} expand - If true, expand the groups. Otherwise, collapse them.
```

Comment:
nit: this is not an established pattern (yet?) of separating the param name from the description with a `-`.  Convince me we should?